go-controller/pkg/node/ovspinning/ovspinning_linux.go

```diff
@@ -0,0 +1,179 @@
+//go:build linux
+// +build linux
+
+package ovspinning
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/fsnotify/fsnotify"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+	"golang.org/x/sys/unix"
+	"k8s.io/klog/v2"
+)
+
+// These variables are meant to be used in unit tests
+var tickDuration time.Duration = 10 * time.Second
+var getOvsVSwitchdPIDFn func() (string, error) = util.GetOvsVSwitchdPID
+var getOvsDBServerPIDFn func() (string, error) = util.GetOvsDBServerPID
+var featureEnablerFile string = "/etc/openvswitch/enable_dynamic_cpu_affinity"
+
+// Run monitors OVS daemon's processes (ovs-vswitchd and ovsdb-server) and sets their CPU affinity
+// masks to that of the current process.
+// This feature is enabled by the presence of a non-empty file in the path `/etc/openvswitch/enable_dynamic_cpu_affinity`
+func Run(stopCh <-chan struct{}) {
+
+	klog.Infof("Starting OVS daemon CPU pinning")
+
+	fileWatcher, err := createFileWatcherFor(featureEnablerFile)
```

Comment:
maybe add a comment here that if the file is missing on start up we do not continue the goroutine